app/models/dialog_field_sorted_item.rb

```diff
@@ -93,7 +93,7 @@ def sort_data(data_to_sort)
   def raw_values
     @raw_values ||= dynamic ? values_from_automate : self[:values].to_miq_a
     unless @raw_values.collect { |value_pair| value_pair[0] }.include?(default_value)
-      self.default_value = sort_data(@raw_values).first.first
+      self.default_value = sort_data(@raw_values).first.try(:first)
```

Comment:
@eclarizio It looks like `@raw_values` is an empty array here for the tag control.  I would expect other field to return a multi-dimensional array (like `[[1, "One"], [2, "Two"]]`) if they had values, otherwise they could have empty arrays as well.  Does that sound correct to you?

This change will now set `default_value` to `nil` when we get an empty `@raw_values` array.